src/frontend/js/components/ContractFrame/OrganizationContractFrame.tsx

```diff
@@ -4,14 +4,21 @@ import { useJoanieApi } from 'contexts/JoanieApiContext';
 import AbstractContractFrame, {
   AbstractProps,
 } from 'components/ContractFrame/AbstractContractFrame';
-import { Contract } from 'types/Joanie';
+import { Contract, CourseProductRelation } from 'types/Joanie';
 
 interface Props extends AbstractProps {
   contractIds?: Contract['id'][];
   organizationId: string;
+  courseProductRelationId?: CourseProductRelation['id'];
```

Comment:
Why not require here an array of id ?
IMO it will prevent a future refactoring.

Reply:
I didn't see any need's of signing multiple training contracts.
We only list things by organization or by training.
Having multiple trainings list and action would throw lot's of refactoring anyway, so i make it to fit our current usages.

Reply:
My thought was to allow an organization to sign contracts to a batch of trainings ?